src/platform/plugins/shared/dashboard/server/content_management/v1/cm_services.ts

```diff
@@ -274,55 +306,49 @@ export const referenceSchema = schema.object(
   { unknowns: 'forbid' }
 );
 
-const dashboardAttributesSchemaResponse = dashboardAttributesSchema.extends({
-  // Responses always include the panel index (for panels) and gridData.i (for panels + sections)
-  panels: schema.arrayOf(
-    schema.oneOf([
-      panelSchema.extends({
-        panelIndex: schema.string(),
-        gridData: panelGridDataSchema.extends({
-          i: schema.string(),
-        }),
-      }),
-      sectionSchema.extends({
-        gridData: sectionGridDataSchema.extends({
-          i: schema.string(),
-        }),
-        panels: schema.arrayOf(
-          panelSchema.extends({
-            panelIndex: schema.string(),
-            gridData: panelGridDataSchema.extends({
-              i: schema.string(),
-            }),
-          })
-        ),
-      }),
-    ]),
-    { defaultValue: [] }
-  ),
+export const dashboardCreateRequestAttributesSchema = schema.object({
+  ...searchResultsAttributes,
+  ...dashboardAdditionalAttributes,
+  references: schema.maybe(schema.arrayOf(referenceSchema)),
+  spaces: schema.maybe(schema.arrayOf(schema.string())),
 });
 
-export const dashboardItemSchema = schema.object(
-  {
-    id: schema.string(),
-    type: schema.string(),
-    version: schema.maybe(schema.string()),
-    createdAt: schema.maybe(schema.string()),
-    updatedAt: schema.maybe(schema.string()),
-    createdBy: schema.maybe(schema.string()),
-    updatedBy: schema.maybe(schema.string()),
-    managed: schema.maybe(schema.boolean()),
-    error: schema.maybe(apiError),
-    attributes: dashboardAttributesSchemaResponse,
-    references: schema.arrayOf(referenceSchema),
-    namespaces: schema.maybe(schema.arrayOf(schema.string())),
-    originId: schema.maybe(schema.string()),
-  },
-  { unknowns: 'allow' }
-);
+export const dashboardUpdateRequestAttributesSchema = schema.object({
+  ...searchResultsAttributes,
+  ...dashboardAdditionalAttributes,
+  type: schema.maybe(schema.string()),
+  references: schema.maybe(schema.arrayOf(referenceSchema)),
+  spaces: schema.maybe(schema.arrayOf(schema.string())),
+});
 
-export const dashboardSearchResultsSchema = dashboardItemSchema.extends({
-  attributes: searchResultsAttributesSchema,
+export const dashboardResponseAttributesSchema = schema.object({
+  ...searchResultsAttributes,
+  ...dashboardAdditionalAttributes,
+  references: schema.maybe(schema.arrayOf(referenceSchema)),
+  spaces: schema.maybe(schema.arrayOf(schema.string())),
+});
+
+export const dashboardAttributesSchemaRequest =
+  dashboardCreateRequestAttributesSchema.extends(dashboardPanels);
+
+export const dashboardAttributesSchema = schema.object({
+  ...searchResultsAttributes,
+  ...dashboardAdditionalAttributes,
+  version: schema.string(),
```

Comment:
```suggestion
```

This `version` is overwriting the `version` already specified in the `dashboardAdditionalAttributes`. We should just remove it here.